files/image_config/rsyslog/rsyslog-container.conf.j2

```diff
@@ -14,8 +14,24 @@ $ModLoad imuxsock # provides support for local system logging
 #
 # Set a rate limit on messages from the container
 #
-$SystemLogRateLimitInterval 300
-$SystemLogRateLimitBurst 20000
+
+{% if SYSLOG_CONFIG_FEATURE is defined %}
+{% if container_name in SYSLOG_CONFIG_FEATURE %}
+{% if 'rate_limit_interval' in SYSLOG_CONFIG_FEATURE[container_name]%}
+{% set rate_limit_interval = SYSLOG_CONFIG_FEATURE[container_name]['rate_limit_interval'] %}
+{% endif %}
+{% if 'rate_limit_burst' in SYSLOG_CONFIG_FEATURE[container_name]%}
+{% set rate_limit_burst = SYSLOG_CONFIG_FEATURE[container_name]['rate_limit_burst'] %}
+{% endif %}
+{% endif %}
+{% endif %}
+
+{% if rate_limit_interval is defined %}
+$SystemLogRateLimitInterval {{ rate_limit_interval }}
+{% endif %}
+{% if rate_limit_burst is defined %}
+$SystemLogRateLimitBurst {{ rate_limit_burst }}
+{% endif %}
```

Comment:
This change means you have unlimited messages unless you turn on the rate-limiting feature?  Is it possible to have rate limiting but not make it configurable?

Reply:
`This change means you have unlimited messages unless you turn on the rate-limiting feature?`
Yes.
`Is it possible to have rate limiting but not make it configurable?`
Default rate-limit configuration is located at files/build_templates/init_cfg.json.j2, check this PR: https://github.com/sonic-net/sonic-buildimage/pull/12490